src/main/java/org/opentripplanner/middleware/tripmonitor/jobs/CheckMonitoredTrip.java

```diff
@@ -799,9 +801,21 @@ public boolean shouldSkipMonitoredTripCheck(boolean persist) throws Exception {
         return true;
     }
 
+    /** Check if the matching itinerary start time is in the future */
+    private boolean isMatchingItineraryStartTimeValid() {
```

Comment:
I think a better name for this method would be `isMatchingItineraryStartTimeInTheFuture()`